entity/entity.py

```diff
@@ -16,3 +33,19 @@ def render(self, screen, camera: Camera):
             )
         )
         screen.blit(self.graphic, rect)
+
+    def _position(self, dt):
+        pos_x += self.velocity_x * dt + .5 * self.accel_x * dt**2
+        pos_y = self.velocity_y * dt + .5 * self.accel_y * dt**2
+        self.position = (pos_x ,pos_y )
+
+    def _accelerate(self, dt):
+        if self.accel == True:
+            self.velocity_x += self.accel_x
+            self.velocity_y += self.accel_y
```

Comment:
This should also take an acceleration rate and update the member acceleration variables. No need for the boolean variable. Callers of this method are commanding the entity to accelerate